diff --git a/public/index.html b/public/index.html
index 183a0fe..c7185cc 100644
--- a/public/index.html
+++ b/public/index.html
@@ -11,6 +11,10 @@
     <!--==================== UNICONS ======================-->
     <link rel="stylesheet" href="https://unicons.iconscout.com/release/v4.0.8/css/line.css">
 
+    <!--==================== FLATICON  ======================-->
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/flat-ui/2.3.0/css/flat-ui.min.css" rel="stylesheet"/> <!--load all styles -->
+
+
     <title>MAViS - My Portfolio Website</title>
   </head>
   <body>
diff --git a/src/components/about/About.jsx b/src/components/about/About.jsx
index 31b965d..a2f23b8 100644
--- a/src/components/about/About.jsx
+++ b/src/components/about/About.jsx
@@ -16,7 +16,7 @@ const About = () => {
                 <div className="about__data">
                     <Info />
 
-                    <p className="about__description">Hi, my name is Mavis, a software engineer with a Master’s degree in Information Technology from RMIT University, Melbourne. I specialize in front-end development and digital marketing, and have successfully led projects that enhanced product interfaces and improved loading speeds. I am passionate about using technology to optimize business processes and customer satisfaction, and excel at integrating user feedback to enhance product adaptability. With strong communication skills in both English and Mandarin, I combine my technical expertise and innovative problem-solving abilities to contribute to business growth and success in dynamic environments.
+                    <p className="about__description">Hi, I'm Mavis, a software engineer with a master’s degree of information technology from RMIT University in Melbourne. I specialise in Front-end Development and Digital Marketing. I've led projects that have polished product interfaces and boosted loading speeds. I'm passionate about leveraging technology to enhance business processes and customer satisfaction. With strong communication skills in both English and Mandarin, I combine my technical expertise and innovative problem-solving abilities to contribute to business growth and success in dynamic environments.
                     </p>
 
                     <div className="buttons-container">
diff --git a/src/components/header/header.css b/src/components/header/header.css
index 6d24b86..cec18c1 100644
--- a/src/components/header/header.css
+++ b/src/components/header/header.css
@@ -4,7 +4,7 @@
     top: 0;
     left: 0;
     z-index: var(--z-fixed);
-    background-color: var(--body-color);
+    background-color: var(--container-color);
 }
 
 .nav {
diff --git a/src/components/home/Data.jsx b/src/components/home/Data.jsx
index 7f4cf2e..b4d2e01 100644
--- a/src/components/home/Data.jsx
+++ b/src/components/home/Data.jsx
@@ -5,7 +5,7 @@ const Data = () => {
     return (
         <div className="home__data">
             <h1 className="home__title">
-                Hello World!
+                Wominjeka
                 <svg
                     width="36"
                     height="36"
diff --git a/src/components/home/Home.jsx b/src/components/home/Home.jsx
index 867038e..52b0bd6 100644
--- a/src/components/home/Home.jsx
+++ b/src/components/home/Home.jsx
@@ -6,9 +6,9 @@ import ScrollDown from "./ScrollDown";
 
 const Home = () => {
     return (
-        <section className="home section" id="home">
+        <section className="home section fade-in" id="home">
             <div className="home__container container grid">
-                <div className="home__content grid">
+                <div className="home__content grid slide-in-up">
                     <Social />
 
                     <div className="home__img"></div>
diff --git a/src/components/home/home.css b/src/components/home/home.css
index 35f1bf8..2d4a08b 100644
--- a/src/components/home/home.css
+++ b/src/components/home/home.css
@@ -26,9 +26,9 @@
 }
 
 .home__title {
-    font-family: "Playfair Display", serif;
+    font-family: "Caveat", cursive;
     font-optical-sizing: auto;
-    font-weight: 500;
+    font-weight: 700;
     /*font-style: italic;*/
     font-size: var(--big-font-size);
     margin-bottom: var(--mb-0-25);
@@ -41,6 +41,7 @@
 }
 
 .home__subtitle {
+    font-family: "Caveat", cursive;
     position: relative;
     font-size: var(--h3-font-size);
     padding-left: 5.4rem;
@@ -73,7 +74,7 @@
     justify-self: center;
     width: 300px;
     height: 300px;
-    animation: profile__animate 8s ease-in-out infinite 1s;
+    animation: profile__animate 4s ease-in-out infinite 1s;
 }
 
 .buttons-container {
@@ -136,6 +137,42 @@
     color: var(--title-color);
 }
 
+@keyframes fadeIn {
+    from {
+        opacity: 0;
+    }
+    to {
+        opacity: 1;
+    }
+}
+
+@keyframes slideInUp {
+    from {
+        transform: translateY(100px);
+        opacity: 0;
+    }
+    to {
+        transform: translateY(0);
+        opacity: 1;
+    }
+}
+
+.fade-in {
+    animation: fadeIn 1s ease-in-out forwards;
+}
+
+.slide-in-up {
+    animation: slideInUp 1s ease-in-out forwards;
+}
+
+.fade-in.visible {
+    animation: fadeIn 1s ease-in-out forwards;
+}
+
+.slide-in-up.visible {
+    animation: slideInUp 1s ease-in-out forwards;
+}
+
 /*=============== BREAKPOINTS ===============*/
 /* For large devices */
 @media screen and (max-width: 992px) {
diff --git a/src/components/qualification/qualification.css b/src/components/qualification/qualification.css
index c63903e..78f37c6 100644
--- a/src/components/qualification/qualification.css
+++ b/src/components/qualification/qualification.css
@@ -102,6 +102,8 @@
     }
   }
 
+  
+
 /*=============== BREAKPOINTS ===============*/
 /* For large devices */
 @media screen and (max-width: 992px) {
diff --git a/src/components/services/Services.jsx b/src/components/services/Services.jsx
index b5b500f..e322baa 100644
--- a/src/components/services/Services.jsx
+++ b/src/components/services/Services.jsx
@@ -9,12 +9,12 @@ const Services = () => {
     };
 
     return (
-        <section className="services section" id="services">
-            <h2 className="section__title">Services</h2>
-            <span className="section__subtitle">What I Offer</span>
+        <section className="services section fade-in" id="services">
+            <h2 className="section__title slide-in-up">Services</h2>
+            <span className="section__subtitle slide-in-up">What I Offer</span>
 
             <div className="services__container container grid">
-                <div className="services__content">
+                <div className="services__content slide-in-up">
                     <div>
                         <i className="uil uil-circle-layer services__icon"></i>
                         <h3 className="services__title">
@@ -63,7 +63,7 @@ const Services = () => {
                     </div>
                 </div>
 
-                <div className="services__content">
+                <div className="services__content slide-in-up">
                     <div>
                         <i className="uil uil-icons services__icon"></i>
                         <h3 className="services__title">Graphic <br /> Designer</h3>
@@ -105,7 +105,7 @@ const Services = () => {
                     </div>
                 </div>
 
-                <div className="services__content">
+                <div className="services__content slide-in-up">
                     <div>
                         <i className="uil uil-border-alt services__icon"></i>
                         <h3 className="services__title">Digital <br /> Marketing</h3>
diff --git a/src/components/services/services.css b/src/components/services/services.css
index dc8f46d..c9aa44c 100644
--- a/src/components/services/services.css
+++ b/src/components/services/services.css
@@ -118,6 +118,43 @@
     visibility: visible;
 }
 
+
+@keyframes fadeIn {
+    from {
+        opacity: 0;
+    }
+    to {
+        opacity: 1;
+    }
+}
+
+@keyframes slideInUp {
+    from {
+        transform: translateY(100px);
+        opacity: 0;
+    }
+    to {
+        transform: translateY(0);
+        opacity: 1;
+    }
+}
+
+.fade-in {
+    animation: fadeIn 1s ease-in-out forwards;
+}
+
+.slide-in-up {
+    animation: slideInUp 1s ease-in-out forwards;
+}
+
+.fade-in.visible {
+    animation: fadeIn 1s ease-in-out forwards;
+}
+
+.slide-in-up.visible {
+    animation: slideInUp 1s ease-in-out forwards;
+}
+
 /*=============== BREAKPOINTS ===============*/
 /* For large devices */
 @media screen and (max-width: 992px) {
diff --git a/src/components/skills/Backend.jsx b/src/components/skills/Backend.jsx
index 6adabe9..62b244f 100644
--- a/src/components/skills/Backend.jsx
+++ b/src/components/skills/Backend.jsx
@@ -1,63 +1,59 @@
 import React from 'react'
 
 const Backend = () => {
-  return (
-
-    <div className="skills__content">
-    <h3 className="skills__title">Backend Developer</h3>
-
-    <div className="skills__box">
-        <div className="skills__group">
-            <div className="skills__data">
-                <i class='bx bx-badge-check' ></i>
-
-                <div>
-                    <h3 className="skills__name">PHP</h3>
-                    <span className="skills__level">Intermediate</span>
-                </div>
-            </div>
-
-            <div className="skills__data">
-                <i class='bx bx-badge-check' ></i>
-
-                <div>
-                    <h3 className="skills__name">Node Js</h3>
-                    <span className="skills__level">Basic</span>
-                </div>
-            </div>
-
-            <div className="skills__data">
-                <i class='bx bx-badge-check' ></i>
-
-                <div>
-                    <h3 className="skills__name">Python</h3>
-                    <span className="skills__level">Intermediate</span>
-                </div>
-            </div>
-        </div>
-
-        <div className="skills__group">
-            <div className="skills__data">
-                <i class='bx bx-badge-check' ></i>
-
-                <div>
-                    <h3 className="skills__name">MySQL</h3>
-                    <span className="skills__level">Intermediate</span>
+    return (
+
+        <div className="skills__content">
+            <h3 className="skills__title">Back-end Developer</h3>
+
+            <div className="skills__box">
+                <div className="skills__group">
+                    <div className="skills__data">
+                        <div>
+                            <h3 className="skills__name">
+                                <i class='bx bxl-php'></i></h3>
+                            <span className="skills__level">Intermediate</span>
+                        </div>
+                    </div>
+                    <div className="skills__data">
+                        <div>
+                            <h3 className="skills__name">
+                                <i class='bx bxl-nodejs' ></i>
+                            </h3>
+                            <span className="skills__level">Basic</span>
+                        </div>
+                    </div>
+                    <div className="skills__data">
+                        <div>
+                            <h3 className="skills__name">
+                                <i class='bx bxl-python' ></i>
+                            </h3>
+                            <span className="skills__level">Basic</span>
+                        </div>
+                    </div>
                 </div>
-            </div>
-
-            <div className="skills__data">
-                <i class='bx bx-badge-check' ></i>
 
-                <div>
-                    <h3 className="skills__name">MongoDB</h3>
-                    <span className="skills__level">Intermediate</span>
-                </div>
-            </div>
+                <div className="skills__group">
+                    <div className="skills__data">
+                        <div>
+                            <h3 className="skills__name">
+                                <i class='bx bxl-mongodb' ></i>
+                            </h3>
+                            <span className="skills__level">Basic</span>
+                        </div>
+                    </div>
+                    <div className="skills__data">
+                        <div>
+                            <h3 className="skills__name">
+                            <i class='bx bxl-github'></i>
+                            </h3>
+                            <span className="skills__level">Intermediate</span>
+                        </div>
+                    </div>
                 </div>
             </div>
         </div>
-  )
+    )
 }
 
-export default Backend
\ No newline at end of file
+export default Backend;
\ No newline at end of file
diff --git a/src/components/skills/Frontend.jsx b/src/components/skills/Frontend.jsx
index d5476e4..0826746 100644
--- a/src/components/skills/Frontend.jsx
+++ b/src/components/skills/Frontend.jsx
@@ -4,68 +4,68 @@ const Frontend = () => {
     return (
         <div className="skills__content">
 
-            <h3 className="skills__title">Frontend developer</h3>
+            <h3 className="skills__title">Front-end Developer</h3>
 
 
             <div className="skills__box">
                 <div className="skills__group">
                     <div className="skills__data">
-                        <i class='bx bx-badge-check' ></i>
-
                         <div>
-
-                            <h3 className="skills__name">HTML</h3>
-                            <span className="skills__level">Basic</span>
-
+                            <h3 className="skills__name">
+                                <i class='bx bxl-html5' ></i>
+                            </h3>
+                            <span className="skills__level">Advanced</span>
                         </div>
                     </div>
 
                     <div className="skills__data">
-                        <i class='bx bx-badge-check' ></i>
 
                         <div>
 
-                            <h3 className="skills__name">CSS</h3>
+                            <h3 className="skills__name">
+                                <i class='bx bxl-css3' ></i>
+                            </h3>
                             <span className="skills__level">Advanced</span>
 
                         </div>
                     </div>
 
                     <div className="skills__data">
-                        <i class='bx bx-badge-check' ></i>
-
                         <div>
 
-                            <h3 className="skills__name">JavaScript</h3>
-                            <span className="skills__level">Intermediate</span>
+                            <h3 className="skills__name"><i class='bx bxl-javascript' ></i></h3>
+                            <span className="skills__level">Advanced</span>
                         </div>
                     </div>
                 </div>
 
                 <div className="skills__group">
                     <div className="skills__data">
-                        <i class='bx bx-badge-check' ></i>
 
                         <div>
-                            <h3 className="skills__name">Bootstrap</h3>
+                            <h3 className="skills__name">
+                                <i class='bx bxl-bootstrap' ></i>
+                            </h3>
                             <span className="skills__level">Intermediate</span>
                         </div>
                     </div>
 
                     <div className="skills__data">
-                        <i class='bx bx-badge-check' ></i>
 
                         <div>
-                            <h3 className="skills__name">Git</h3>
-                            <span className="skills__level">Intermediate</span>
+                            <h3 className="skills__name">
+                                <i class='bx bxl-vuejs' ></i>
+                            </h3>
+                            <span className="skills__level">Basic</span>
                         </div>
                     </div>
 
                     <div className="skills__data">
-                        <i class='bx bx-badge-check' ></i>
 
                         <div>
-                            <h3 className="skills__name">React</h3>
+                            <h3 className="skills__name">
+                            <i class='bx bxl-react' ></i>
+                            </h3>
                             <span className="skills__level">Intermediate</span>
                         </div>
                     </div>
diff --git a/src/components/skills/skills.css b/src/components/skills/skills.css
index 2ce5462..ae09526 100644
--- a/src/components/skills/skills.css
+++ b/src/components/skills/skills.css
@@ -15,39 +15,51 @@
     font-size: var(--normal-font-size);
     font-weight: var(--font-medium);
     text-align: center;
-    margin-bottom: var(--mb-1-5);
+    margin-bottom: var(--mb-1-2);
 }
 
 .skills__box {
     display: flex;
     justify-content: center;
-    column-gap: 2.5rem;
+    column-gap: 3rem;
+    margin-bottom: 1.2rem;
 }
 
 .skills__group {
     display: grid;
     align-items: flex-start;
-    row-gap: 1rem;
+    row-gap: 0.8rem;
+    align-items: center;
 }
 
-.skills_data {
+.skills__data {
     display: flex;
-    column-gap: 0.5rem;
+    column-gap: 1rem;
+    align-items: center;
 }
 
-.skills .bx-badge-check {
-    font-size: 1rem;
-    color: var(--title-color);
+.skills .bx {
+    font-size: 3.5rem;
+    color: purple;
+    align-items: center;
 }
 
 .skills__name {
     font-size: var(--normal-font-size);
     font-weight: var(--font-medium);
-    line-height: 18px;
+    align-items: center;
 }
 
 .skills__level {
-    font-size: var(--tiny-font-size);
+    font-size: var(--smaller-font-size);
+    line-height: 0.2rem; /* 调整行高以缩小行间距 */
+    align-items: center;
+}
+
+.skills__data > div {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
 }
 
 /*=============== BREAKPOINTS ===============*/
